pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -143,10 +143,11 @@ type AzVolumeList struct {
 // AzVolumeAttachment is a specification for a AzVolumeAttachment resource
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Namespaced
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="The age of the attachment"
```

Comment:
This is the line that adds the Age column.

Reply:
🥳